Userland/Services/SystemServer/ClientConnection.cpp

```diff
@@ -56,6 +56,18 @@ OwnPtr<Messages::ServiceManagementServer::GreetResponse> ClientConnection::handl
     return make<Messages::ServiceManagementServer::GreetResponse>();
 }
 
+OwnPtr<Messages::ServiceManagementServer::ServiceListResponse> ClientConnection::handle(const Messages::ServiceManagementServer::ServiceList&)
+{
+    // FIXME: Provide information about status
+    auto& services = ServiceManagement::the().services();
+    Vector<String> service_names;
+
+    for (auto& service : services)
+        service_names.append(String::formatted("{}{}", service.is_running() ? "+" : "-", service.name()));
```

Comment:
This, too, formats info server-side; even after you've switched to JSON in the other place.